labels/en-US.json

```diff
@@ -379,8 +379,8 @@
   "mobileOrderConfirmationMessage": "{0} Order Alerts. We will text you when your order status changes. Msg freq may vary. Reply {1} to opt out. Msg&Data rates may apply",
   "mobileOrderOptInMessage": "Thank you for opting in to SMS alerts for your Curbside order. We will notify you when your order is ready for pickup.",
   "mobilePickLocation": "Pickup from/Ship from:",
-  "mobilePickupReminder":"Reminder - Items in your order {0} are ready for pickup. Review the order at {1}",
-  "mobileReadyForPickup":"Items in your order {0} are ready for pickup at {1}. Review the order at {2}",
+  "mobilePickupReminder": "Reminder - Items in your order {0} are ready for pickup. Review the order at {1}",
+  "mobileReadyForPickup": "Items in your order {0} are ready for pickup at {1}. Review the order at {2}",
```

Comment:
Don't make changes to labels you did not touch. This is core theme and our clients have to merge this. Revert all unnecessary whitespace changes in both label files.

Reply:
done.